cmd/csaf_downloader/downloader.go

```diff
@@ -281,6 +281,13 @@ func (d *downloader) downloadFiles(label csaf.TLPLabel, files []csaf.AdvisoryFil
 			continue
 		}
 
+		// Warn if we do not get JSON.
+		if ct := resp.Header.Get("Content-Type"); ct != "application/json" {
+			log.Printf(
+				"WARN: The content type of %s should be 'application/json but is '%s'\n",
```

Comment:
The quotes around application/json are not closed. ('application/json -> 'application/json' )